libraries/ae-inventory-processor/src/main/resources/META-INF/templates/inventory-report-vulnerability/vulnerability-report.properties.vt

```diff
@@ -113,3 +113,19 @@ ae.vulnerability.advisor.time.step.first.de=$time.getDe()
 ae.vulnerability.advisor.time.step.latest.timestamp=$time.getTime()
 ae.vulnerability.advisor.time.step.latest.en=$time.getEn()
 ae.vulnerability.advisor.time.step.latest.de=$time.getDe()
+
+
+# may be 'sbom-importer', 'cyclonedx-importer' or 'merged'
+#set($creator = $vulnerabilityAdapter.getInventoryCreator())
+ae.inventory.process.creator.name=$creator.getName()
+ae.inventory.process.creator.timestamp=$creator.getTime()
+
+
+# may be 'inventory-enricher' or 'advisor-periodic-enrichment'
+#set($enricher = $vulnerabilityAdapter.getInventoryCreator())
+ae.inventory.process.enricher.name=$enricher.getName()
```

Comment:
In my eyes this wont work; how would you handle multiple enrichers?

I would propose concrete properties for destinct processing stages.

Please also consider how a consumer would use those properties. There will not be much logic. A consumer will use a specific property for a specific concern.